Tensile/SolutionStructs.py

```diff
@@ -4111,7 +4120,8 @@ def assignDerivedParameters(state):
     # check LocalDotLayout
     ldl = state["LocalDotLayout"]
     if ldl> 1:
-      state["DirectToLds"] = False
+      state["DirectToLdsA"] = False
+      state["DirectToLdsB"] = False
```

Comment:
Same here?

Reply:
I quickly tried LDL>1 + DTL=1, but I could not enable the combination....
It should be same as BL=0 and DTL=1.
Tensile will not generate duplicated kernels if all kernel parameters are the same (DTL1 is converted to DTL0 by this line).